README.md

```diff
@@ -1,26 +1,29 @@
-# What is Hyperledger Iroha?
+# Welcome!
+
+## What is Hyperledger Iroha?
 
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/960/badge)](https://bestpractices.coreinfrastructure.org/projects/960)
-[![Snap Status](https://build.snapcraft.io/badge/hyperledger/iroha.svg)](https://build.snapcraft.io/user/hyperledger/iroha)
 [![Build Status](https://jenkins.soramitsu.co.jp/buildStatus/icon?job=iroha/iroha-hyperledger/master)](https://jenkins.soramitsu.co.jp/job/iroha/job/iroha-hyperledger/job/master/)
 [![Throughput Graph](https://graphs.waffle.io/hyperledger/iroha/throughput.svg)](https://waffle.io/hyperledger/iroha/metrics/throughput)
 
-Blockchain platform Hyperledger Iroha is designed for simple creation and management of assets. This is a distributed ledger of transactions.
+Iroha is a straightforward DLT, inspired by Japanese Kaizen principle — eliminate excessiveness (muri). Iroha has essential functionality for your asset, information and identity management needs, at the same time being an efficient and trustworthy byzantine fault-tolerant tool for your enterprise needs. 
 
-Check [overview](http://iroha.readthedocs.io/en/latest/overview.html) page of our documentation.
+Check the [overview](http://iroha.readthedocs.io/en/latest/overview.html) page of our documentation.
+[Here](https://www.youtube.com/channel/UCYlK9OrZo9hvNYFuf0vrwww) is a YouTube channel where we upload meetings and explanatory videos - check them out! 
 
 <img height="300px" src="docs/image_assets/Iroha_3_sm.png"
  alt="Iroha logo" title="Iroha" align="right" />
 
 Iroha has the following features:
-1. Creation and management of custom complex assets, such as currencies or indivisible rights, serial numbers, patents, etc.
+1. Creation and management of custom assets, such as currencies or indivisible rights, serial numbers, patents, etc.
```

Comment:
```suggestion
1. Creation and management of custom fungible assets, such as currencies, kilos of gold, etc.
```
We don't have the support of non-fungible tokes like `indivisible rights, serial numbers, patents`.

If you refer to SetAccountDetail, then it does not have compare-and-swap access mechanism, thus it cannot be considered as a way to implement the support for `indivisible rights, serial numbers, patents`.